eduiddashboard/utils.py

```diff
@@ -391,7 +388,7 @@ def _safe_clean(untrusted_text, strip_characters=False):
     :rtype: str | unicode
     """
     try:
-        return clean(untrusted_text, strip=strip_characters)
+        return untrusted_text
```

Comment:
This does not seem good, to just return untrusted_text. Is there a problem with bleach?